src/ILCompiler.Compiler/src/Compiler/UsageBasedMetadataManager.cs

```diff
@@ -573,5 +596,11 @@ public enum UsageBasedMetadataGenerationOptions
         /// Scan IL for common reflection patterns to find additional compilation roots.
         /// </summary>
         ILScanning = 4,
+
+        /// <summary>
+        /// Specifies that all types and methods in user assemblies should be considered dynamically
+        /// used.
+        /// </summary>
+        FullUserAssemblyRooting,
```

Comment:
```suggestion
        FullUserAssemblyRooting = 8,
```

?

Reply:
Oh yeah, Good catch. That would be confusing.